openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -170,9 +167,14 @@ def create_account_with_params(request, params):
     third_party_auth_credentials_in_api = 'provider' in params
     is_third_party_auth_enabled = third_party_auth.is_enabled()
 
-    if is_third_party_auth_enabled and (pipeline.running(request) or third_party_auth_credentials_in_api):
```

Comment:
you are replacing `third_party_auth_credentials_in_api`  with `'provider' in request.POST` in your `is_sso_request` method, is that intentional? if so, why?

Reply:
`third_party_auth_credentials_in_api = 'provider' in params` In this line **params is equal to request.POST**  and this logic is used in multiple places so it was moved to ` is_sso_request()`